spec/spec.md

```diff
@@ -156,6 +157,21 @@ and so on.
 
 An error is reported if any `Tx` is `void`.
 
+### Tuple Elimination
+
+```grammar
+expr += (TupleAccess tup:<expr> index:<int_val>)
+```
+
+Retrieves the value from the `index`-th position of the tuple.
+
+Let `T` be the type of `tup`. If `T` is not a `tuple` type, an error is
+reported. If `index` is an integer value less than 0, or greater than or equal
+to the number of types in the tuple type `T`, an error is reported.
```

Comment:
```suggestion
to the number of positions in the tuple type `T`, an error is reported.
```

I'm suggesting `positions` because it's possible the reader might thing `types` is a set, so `(int, int)` would have 2 `positions`, but 1 `types`.